docs/source/config_file.rst

```diff
@@ -57,6 +57,7 @@ section names in square brackets and flag settings of the form
   .. _config-precedence:
 
   When options conflict, the precedence order for configuration is:
+
```

Comment:
This newline is not related to this PR's topic, but it fixes a small rendering issue. Right now on current master, the list is rendered like this:

![image](https://user-images.githubusercontent.com/4455652/68077037-01104300-fdbd-11e9-8e9a-3c7d865d5fef.png)

This is because the list items are too deeply indented. With a proper separation of the list and the preceding paragraph, it becomes

![image](https://user-images.githubusercontent.com/4455652/68077058-7e3bb800-fdbd-11e9-978e-40bff5d54e64.png)